src/celestia/celestiacore.h

```diff
@@ -422,11 +445,15 @@ class CelestiaCore // : public Watchable<CelestiaCore>
     std::shared_ptr<TextureFont> font{ nullptr };
     std::shared_ptr<TextureFont> titleFont{ nullptr };
 
+    class TextPrintPosition
```

Comment:
please move into .h file together with AbsoluteTextPrintPosition and RelativeTextPrintPosition